### ISSUE TITLE
Name is not defined for "Like" button of any comment.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-iOS-SDK#704]
Category: Screen Reader
MAS Rule: MAS4.2.1
Repro Steps
1. Launch Embedded Social Test App.
2. Navigate to Hamburger button and double tap.
3. Navigate to Popular link and double tap.
4. Popular page will be appear.
5. Navigate  to the comment button of any topic and double tap.
6. Comment on topic and double tap on post button.
7. Navigate the page via swipe gesture..
8. Verify that name is defined for like button or not.
Expected:
Name should be defined for like button of any comment as like it is defined for comment button of any reply of comment. So that screen reader announce properly.
Actual:
Name is not defined for like button of any comment. But when user reply on any comment then name is defined for like button.

Note: 
1. Same issue is repro for Upload image button, initially voice over announcing place holder no image button, but after the selecting image it announce button only.
2. Name is not defined for Grid view button, Image button and more button on home page and popular page.
3. Name is also not defined for Upload image button on create profile page.
4. Voice over is reading more button two times as button button.
Test Environment:
OS: iOS 11.0.2
App: Embedded Social Test
App Version: 0.7.8 (8)
Tools: Voice Over, Keyboard
